plugins/SitesManager/API.php

```diff
@@ -411,13 +411,18 @@ private function getSitesFromIds($idSites, $limit = false)
 
     protected function getNormalizedUrls($url)
     {
-        if (strpos($url, 'www.') !== false) {
-            $urlBis = str_replace('www.', '', $url);
-        } else {
-            $urlBis = str_replace('://', '://www.', $url);
-        }
-
-        return array($url, $urlBis);
+        // if found, remove scheme and www. from URL
+        $hostname = str_replace('www.', '', $url);
+        $hostname = str_replace('http://', '', $hostname);
+        $hostname = str_replace('https://', '', $hostname);
+
+        // return all variations of the URL
+        return array(
+            "http://" . $hostname,
```

Comment:
Is it on purpose that we only allow http/https? Eg in mobile apps there could be URLs like `fb://profile/33138223345` see http://en.wikipedia.org/wiki/Mobile_deep_linking Probably people wouldn't enter such URLs but maybe they do? Maybe we could always include the original `$url` as well` Seems a bit restrictive otherwise